docker-compose.yml

```diff
@@ -139,10 +142,13 @@ services:
     command: hupper -m celery -A warehouse worker -B -S redbeat.RedBeatScheduler -l info
     volumes:
       - ./warehouse:/opt/warehouse/src/warehouse:z
+      - packages:/var/opt/warehouse/packages
+      - packages-archive:/var/opt/warehouse/packages-archive
     env_file: dev/environment
     environment:
       C_FORCE_ROOT: "1"
       FILES_BACKEND: "warehouse.packaging.services.LocalFileStorage path=/var/opt/warehouse/packages/ url=http://files:9001/packages/{path}"
+      ARCHIVE_FILES_BACKEND: "warehouse.packaging.services.LocalArchiveFileStorage path=/var/opt/warehouse/packages-archive/ url=http://files:9001/packages-archive/{path}"
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏is this still necessary if it's been added to `dev/environment`?

Reply:
maybe not. but note the URL is configured to seek files from the files host, not localhost

If the worker needs to fetch a file to pretend to be "real" it needs to hit the right url.

Reply:
in summary: its not the same config as is in dev/environment

Reply:
I must have missed that, sorry!  Nothing to see here, move along.